pygmt/src/info.py

```diff
@@ -30,8 +30,8 @@ def info(table, **kwargs):
     and *dy* are needed). If the ``per_column`` parameter is combined with
     ``spacing``, then the numpy.ndarray output will be rounded up/down for as
     many columns as there are increments provided in ``spacing``. A similar
-    parameter ``nearest_multiple`` option will provide a numpy.ndarray in the
-    form of [*zmin*, *zmax*, *dz*]`` for makecpt.
+    parameter ``nearest_multiple`` will provide a numpy.ndarray in the form
+    of ``[zmin, zmax, dz]`` for makecpt.
```

Comment:
Italics make more sense to me.
```suggestion
    of [*zmin*, *zmax*, *dz*] for makecpt.
```